### PR TITLE
feat: dompurify as chunk instead of lazy

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Instead of lazy loading dompurify as a js assets, this PR adds it as a chunked library. We are now using the sanitizer in more places, the probability that the user will have to download it is bigger, therefore the explicit lazy loading isn't that necessary anymore and we can rely on chunks. In addition, it makes the code imperative which can be easier to introduce the component `<HTML />` we briefly commented in one of my last PR.

# Changes

- replace dompurify js assets with dependency
- adapt import and sanitizer function accordingly
- update jest to make dompurify supported for nodejs tests

### CODE REVIEW

**Comment by peterpeterparker** [comment]
what ya think @mstrasinskis ?

File: frontend/src/lib/utils/html.utils.ts [hunk 1]
```diff
@@ -65,29 +66,16 @@ export const markdownToHTML = async (): Promise<(text: string) => string> => {
...
-  return convertMarkdownToHTML(sanitizeText(text ?? ""));
+  const convertMarkdownToHTML = await markdownToHTML();
+  return convertMarkdownToHTML(sanitize(text ?? ""));
 };
 
-export const sanitizeHTML = async (text: string): Promise<string> => {
-  const sanitizeText = await sanitize();
-  return sanitizeText(text);
+export const sanitize = (text: string): string => {
+  const isBrowser = typeof window !== "undefined";
```
Comment (mstrasinskis) [inline frontend/src/lib/utils/html.utils.ts:79]: Not against the current solution, just a reminder that there is `isNode` function available ;)

**Review by mstrasinskis — APPROVED** [review]
LGTM. Thank you, totally makes sense.

File: frontend/src/lib/utils/html.utils.ts (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/utils/html.utils.ts]: Ah excellent! Replaced `isBrowser` with `!isNode` thanks!

**Pull request merged**